### DISCUSSION TITLE
Mensaje de Commit correcto

### DISCUSSION THREAD

**Discussion by MarioArocaPaez** [perealberto/decide-single-zambrano#89] [Q&A]
Buenas chicos, he estado echando un ojo a todos los commits que hay por el repo. Os recuerdo que este es nuestro tipo de mensajes ahora mismo:

tipo: asunto #id

cuerpo

### tipo
# feat (nueva funcionalidad)
# fix (corrección de bugs)
# research (incorporación de código experimental, puede ser no funcional)
# refactor (refactorización de código)
# docs (actualización de documentación)
# test (incorporación o modificación de tests)
# conf (modificación de archivos de configuración)

### asunto
# Consiste en una breve descripción del problema que se ha tratado y que debe de comenzar con un verbo en participio. 
# Se referenciará la issue correspondiente (en caso de existir) tal que: `#<ID_issue>`

### cuerpo (opcional)
# Se utilizará en caso de que el asunto no sea suficientemente descriptivo.

### Ejemplo
# conf : Actualizado docker-compose.yml #1

De aquí he estado viendo que @silvglad los hace un poco distintos
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/71765de5-8772-426f-bf50-63d62880206a)
Es cierto que al principio hablamos de poner entre paréntesis el modulo en el que trabajamos, pero nunca llegamos a hacerlo ni ponerlo en el gitmessage pero Enrique ha estado haciéndolo.
Esto es un problema porque estamos 4 personas haciendo un tipo de mensaje y 1 otro, entonces ¿Qué creéis que deberíamos hacer? 
Yo creo que debería de quedar registrado que esto ha pasado y que Enrique pase a hacer los mensajes como el resto del equipo, no sé como lo veis.

**Comment by silvglad** [comment]
A unas malas se puede modificar nuestro documento de commits para incluir que, opcionalmente, se puede incluir los paréntesis para indicar en cuál módulo se está trabajando. No obstante, está bien que quede constancia al respecto en esta discusión y me aseguraré de realizar los mensajes de commits como vosotros

**Comment by laurolmer** [comment]
Debemos utilizar todos la misma estructura en mi opinión, porque si no se vuelve un poco caótico

**Reply by perealberto** [reply]
Coincido con nuestra compañera @laurolmer. Creo que deberíamos seguir la plantilla de commit de la forma más fiel posible para asegurar la uniformidad en la documentación y las incidencias del proyecto.
